docs/semgrep-cloud-platform/github-pr-comments.md

```diff
@@ -56,4 +57,21 @@ Only rules in the **Comment** and **Block** columns of your [Rule board](https:/
 
 <EnableAutofix />
 
+## Dataflow traces in PR comments
+
+![Screenshot of a GitHub PR comment with dataflow traces](/img/dataflow-traces-pr-comments.png#bordered)
+**Figure** An inline GitHub pull request comment with dataflow traces.
+
+<DisplayTaintedDataIntro />
+
+### Viewing the path of tainted data in PR comments
+
+To enable dataflow traces feature in your CI pipeline, fulfill the following prerequisites:
+
+:::info Prerequisites
+- Enable GitHub pull request comments from Semgrep.
```

Comment:
Dataflow trace graphs will also work in gitlab PR comments, just not bitbucket

Reply:
Ok! I will update the PR accordingly.

Reply:
A section about GitLab MR comments added. We might need to add a screenshot of dataflow traces in GitLab.